docs/content/Modpacks/Other-Topics/Spoilage.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Spoilage
+---
+
+**Spoilage** is a mechanic that allows items to, well, *spoil*.\
+Spoilable items spoil based on the amount of ticks that passed from their creation,
+or, more specifically, from one of these events (due to Minecraft's limitations):
+ - The item was crafted in a GregTech recipe
+ - The item was crafted in a crafting table
+ - The item was in a GregTech inventory at any point in time
+ - The item was in a player's inventory for at least 1 tick
+ - The item was dropped
+ - `ISpoilableItem.update(ItemStack, null)` was called
+
+If you want to make an item spoil, you can either make it implement the `ISpoilableItem` interface, or
+attach an `ISpoilableItem` to it. Here's some examples:
+```java
+public class Example {
+    public void attachSpoilables() {
+        // make diamonds spoil into dirt in 100 seconds
+        new SpoilableBehaviour(Items.DIRT, 20*100).attachTo(Items.DIAMOND);
```

Comment:
Maybe make the comments have capital letters and periods in our docs. I think that's how we do it else where too, not 100% sure tho